components/HamburgerMenu.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <div class="cursor-pointer w-full flex" v-close-on-outside-click>
+        <SVGHamburgerMenuIcon id="hamburger-menu-icon" @click="toggleMenu()" alt="hamburger menu" class="h-8 w-8" />
+        <div id="hamburger-menu" v-show="isMenuOpen"
+            class="z-20 absolute top-0 left-0 h-full w-2/3 pt-6 pb-8 rounded bg-primary-bg border-2 flex flex-col justify-between">
+            <div id="upper-section">
+                <div id="header" class="flex justify-between px-5">
```

Comment:
Nit: I think we should make the div ids more descriptive. What do you think about something like `hamburger-menu-header` instead?

Reply:
good observation. So in Vue, you can reuse the same ids in components as their name will change when being imported and the parent grouping is added so you don't have to prefix your component. like `header` becomes `hamburger-menu-header` when it's in the `hamburger-menu` component. 

butttttt.....looks like our compiler isn't doing this anymore?? 🤔 I blame nuxt 🙄 .  lemme see if we can turn it back on otherwise I'll add the prefix